dotCMS/src/main/java/com/dotmarketing/portlets/templates/design/bean/TemplateLayoutColumn.java

```diff
@@ -1,50 +1,54 @@
 package com.dotmarketing.portlets.templates.design.bean;
 
 import com.dotmarketing.portlets.templates.design.util.PreviewTemplateUtil;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'com.dotmarketing.portlets.templates.design.util.PreviewTemplateUtil'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)